packages/preview-middleware-client/test/unit/adp/controllers/ControllerExtension.controller.test.ts

```diff
@@ -0,0 +1,300 @@
+import type Dialog from 'sap/m/Dialog';
+import type Event from 'sap/ui/base/Event';
+import type UI5Element from 'sap/ui/core/Element';
+import JSONModel from 'sap/ui/model/json/JSONModel';
+import type RuntimeAuthoring from 'sap/ui/rta/RuntimeAuthoring';
+
+import { fetchMock, sapCoreMock } from 'mock/window';
+
+import ControllerExtension from '../../../../src/adp/controllers/ControllerExtension.controller';
+
+describe('ControllerExtension', () => {
+    beforeAll(() => {
+        fetchMock.mockResolvedValue({
+            json: jest.fn().mockReturnValue({ controllers: [] }),
+            text: jest.fn(),
+            ok: true
+        });
+    });
+
+    describe('onInit', () => {
+        afterEach(() => {
+            jest.restoreAllMocks();
+        });
+
+        test('fills json model with data', async () => {
```

Comment:
there is no check (`expect`) in this test

Reply:
Fixed in: https://github.com/SAP/open-ux-tools/pull/1275/commits/1aa4fddcaaf9dbee692fff79c11a7b1ad11dad22